File: backendworker/NHSOnline.Backend.Worker/GpSystems/Suppliers/Tpp/Models/LinkAccountReply.cs
using System;
using System.Xml.Serialization;

namespace NHSOnline.Backend.Worker.GpSystems.Suppliers.Tpp.Models
{
    [Serializable]
    public class LinkAccountReply
    {
        [XmlAttribute("passphrase")]
        public string Passphrase { get; set; }

        [XmlAttribute("uuid")]
        public Guid Uuid { get; set; }
    }
}

File: bddtests/src/main/kotlin/mocking/tpp/models/LinkAccountReply.kt
package mocking.tpp.models

import javax.xml.bind.annotation.XmlAccessType
import javax.xml.bind.annotation.XmlAccessorType
import javax.xml.bind.annotation.XmlAttribute
import javax.xml.bind.annotation.XmlRootElement

@XmlRootElement(name = "LinkAccountReply")
@XmlAccessorType(XmlAccessType.FIELD)
data class LinkAccountReply (
        @XmlAttribute var passphrase: String = "default passphrase",
        @XmlAttribute var uuid: String = "default uuid"
)
